packages/office-addin-manifest/test/test.ts

```diff
@@ -623,27 +624,56 @@ describe("Unit Tests", function() {
       });
     });
   });
+  describe("export.ts", function() {
+    describe("exportMetadataPackage()", function() {
+      it("export manifest", async function() {
+        this.timeout(6000);
+        const testFolder = path.resolve("./testExecution");
```

Comment:
does this dir contain something? Sometimes it is hard to push dir to git.
If you need to push an empty dir, using `.gitkeep` can solve it.

Reply:
This is just a directory created for testing.  You'll see the other test copies test file to a folder like this that it modifies in the process of the test.  It's just a simple way using temporary disk files that get cleaned up at the end of the test.